packages/flat-components/src/components/ClassroomPage/Timer/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useRef, useState, useEffect, useCallback } from "react";
+import classnames from "classnames";
+import { useTranslation } from "react-i18next";
+import "./style.less";
+import { useIsUnMounted } from "../../../utils/hooks";
+import { RoomStatus } from "../../../types/room";
+import { intervalToDuration } from "date-fns/esm";
+
+export type TimerProps = {
+    roomStatus: RoomStatus;
+    beginTime: number;
+};
+
+const paddingZero = (number?: number): string => {
+    return String(number).padStart(2, "0");
+};
+
+const renderTime = ({ hours, minutes, seconds }: Duration): string => {
+    return hours && hours > 0
+        ? `${paddingZero(hours)}:${paddingZero(minutes)}:${paddingZero(seconds)}`
+        : `${paddingZero(minutes)}:${paddingZero(seconds)}`;
```

Comment:
```suggestion
    return hours > 0
        ? `${paddingZero(hours!)}:${paddingZero(minutes!)}:${paddingZero(seconds!)}`
        : `${paddingZero(minutes!)}:${paddingZero(seconds!)}`;
```

`hours` / `minutes` / `seconds`  must be `number`. see: https://github.com/date-fns/date-fns/blob/54ec0a5e5ca3f472085d3b56be30d801b90829ac/src/intervalToDuration/index.ts#L38

this is `date-fns` bug

Reply:
see: https://github.com/date-fns/date-fns/pull/2917

Reply:
https://github.com/date-fns/date-fns/issues/2891#issuecomment-1003070337